core/domain/stats_domain.py

```diff
@@ -63,13 +66,139 @@
 MAX_ANSWER_DETAILS_BYTE_SIZE = 10000
```

Comment:
```suggestion
MAX_ANSWER_DETAILS_BYTE_SIZE: Final = 10000
```
Please add final to the lines where constants are defined (constants - which are defined with all capitals) 
Also, We use Final to tell MyPy that this value is not going to be changed and it is defined only for reading purpose.

Reply:
Done